motorlib/enums/inhibitedEnds.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
+# Python 3.11 supports `StrEnum` that would make this a bit more concise to write
+# https://docs.python.org/3/library/enum.html#enum.StrEnum
+class InhibitedEnds(str, Enum):
```

Comment:
I think we should try to take this opportunity to make a migration that converts "top"->"forward" and "bottom"->"aft".

Reply:
Sorry for the delay! Here's a kind of silly example that uses all 4 inhibitor configurations:
[example.ric.txt](https://github.com/reilleya/openMotor/files/11889334/example.ric.txt)
and what it looks like on my end, so you can verify nothing changed:
![image](https://github.com/reilleya/openMotor/assets/9787344/192eb371-f096-4480-9d5a-718f3acd6b3f)